parlai/crowdsourcing/tasks/turn_annotations_static/run.py

```diff
@@ -21,13 +21,12 @@
 
 TASK_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
 
+# To run the task with your own config outside this folder (recommended!)
+# Use the command formulation below. For more info,
+# check the README in parlai/crowdsourcing/
+# python turn_annotations_static/run.py conf=<conf name sans yaml> --config-dir <path to directory with a conf/ folder>
```

Comment:
Hmm - in general I'm trying not to repeat the same documentation in both `run.py` and the README, because if we change the command for how to run this later on (which we will), we might forget to update the documentation in both places. Lemme know if you feel strongly about this, and we can discuss further

Reply:
I think I had mentioned to you, but I think we should remove the README. When you open the run.py, you expect tot have instructions there.